src/org/zaproxy/zap/extension/ascanrulesBeta/ZapAddOn.xml

```diff
@@ -6,6 +6,10 @@
 	<url/>
 	<changes>
 	<![CDATA[
+	Minor code changes.<br>
+	Change scanners to honour the technologies enabled (Issue 1618).<br>
+	Added Buffer Overflow scanner to beta (Issue 1605).<br>
```

Comment:
Right now I am just re-pulling from the msrader/zap-extensions.

On Fri, Aug 28, 2015 at 5:40 AM, thc202 <notifications@github.com> wrote:

> In src/org/zaproxy/zap/extension/ascanrulesBeta/ZapAddOn.xml
> <https://github.com/zaproxy/zap-extensions/pull/27#discussion_r38188560>:
>
> > @@ -6,6 +6,10 @@
> >  	<url/>
> >  	<changes>
> >  	<![CDATA[
> > +	Minor code changes.<br>
> > +	Change scanners to honour the technologies enabled (Issue 1618).<br>
> > +	Added Buffer Overflow scanner to beta (Issue 1605).<br>
>
> Oh, what does git status show? upstream/beta?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zaproxy/zap-extensions/pull/27/files#r38188560>.
>